manual/sphinx/user_docs/bout_options.rst

```diff
@@ -272,8 +272,10 @@ and problem.
 Differencing methods
 --------------------
 
-Differencing methods are specified in three section (``[ddx]``,
-``[ddy]`` and ``[ddz]``), one for each dimension.
+Differencing methods are specified in the section (``[ddx]``,
```

Comment:
These sections are `[mesh:ddx]`, `[mesh:ddy]` and `[mesh:ddz]` now, would be good to update the documentation here. I guess also `[mesh:diff]`?